public/src/components/channelManagement/bannerTests/utils/defaults.ts

```diff
@@ -30,6 +33,7 @@ const PROD_DEFAULT_VARIANT: BannerVariant = {
   template: BannerTemplate.ContributionsBanner,
   bannerContent: {
     messageText: '',
+    paragraphs: [''],
```

Comment:
Need to check to see if setting paragraphs to an empty array would be the better thing to do here.

Reply:
I would go with empty array, and remove `messageText` entirely

Reply:
Done